src/settings/Settings.js

```diff
@@ -114,19 +114,18 @@ export const SETTINGS = {
         supportedLevels: LEVELS_FEATURE,
         default: false,
     },
-    "feature_cider_composer": {
-        isFeature: true,
-        displayName: _td("Use the new, faster, but still experimental composer " +
-            "for writing messages (requires refresh)"),
-        supportedLevels: LEVELS_FEATURE,
-        default: false,
-    },
     "feature_many_integration_managers": {
         isFeature: true,
         displayName: _td("Multiple integration managers"),
         supportedLevels: LEVELS_FEATURE,
         default: false,
     },
+    "useCiderComposer": {
+        isFeature: true,
```

Comment:
should this still be a labs setting given you've listed it to be accessible in COMPOSER_SETTINGS